kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaView.java

```diff
@@ -20,9 +20,8 @@ public SchemaView waitUntilScreenReady() {
     }
 
     @Step
-    public SchemaView isCompatibility(CompatibilityLevel.CompatibilityEnum compatibility){
-        $x("//div//p[.='" + compatibility.getValue() + "']").shouldBe(Condition.visible);
-        return this;
+    public boolean isCompatibility(CompatibilityLevel.CompatibilityEnum compatibility){
+        return    $x("//div//p[.='" + compatibility.getValue() + "']").is(Condition.visible);
```

Comment:
1. extra spaces
2. really don't understand what does this method do - isCompatibility what?? Equals? Visible? imho (i didn't dive into context) we need to assert Compatibility value, so method should be getCompatibility() and return it's value that will be handlet in test using Assert.Equals.. if i'm not right let's discuss

Reply:
Fixed